pkgs/tools/graphics/ploticus/default.nix

```diff
@@ -62,6 +63,14 @@ stdenv.mkDerivation rec {
     ln -s "pl" "$out/bin/ploticus"
   '';
 
+  passthru.tests = {
+    simple = runCommand "${pname}-test" {} ''
+      # trivial test to see if the prefab path munging works
+      mkdir $out/
+      ${ploticus}/bin/pl pl -prefab scat inlinedata="A 1 2" x=2 y=3 -png -o $out/out.png
```

Comment:
Might be nicer to use `finalAttrs.finalPackage` so it works even when the package is overridden. Example: https://github.com/NixOS/nixpkgs/commit/6bce3fc64efd39e7e6b6cc2917eb533afbe33719

Reply:
Thanks!